README.md

```diff
@@ -45,10 +44,17 @@ BLOG_CONFIG = {
     "TAG_NAME": "TAG NAME FOR GENERATING A FEED",
 }
 ```
-- You can now use the data from the blog. To display it the module expects templates at `blog/index.html`, `blog/article.html` and `blog/blog-card.html`. Inspiration can be found at https://github.com/canonical-websites/jp.ubuntu.com/tree/master/templates/blog.
-
 - Run your project and verify that the blog is displaying at the path you specified (f.e. '/blog')
 
+## Templates
+
+- You can now use the data from the blog. To display it the module expects templates at `blog/index.html`, `blog/article.html`, `blog/blog-card.html` . Inspiration can be found at https://github.com/canonical-websites/jp.ubuntu.com/tree/master/templates/blog.
```

Comment:
As mentioned below, I think we should move the URL definitions to www.ubuntu.com, and so this block should probably instead describe how to implement the `group` view.

Reply:
I have updated the module now. It takes a slug and template path and can be configured by the consuming app.